internal/commands/document_template.go

```diff
@@ -26,7 +26,9 @@ const docTemplate = `# Policies
 
 **Parameters:**
 
-{{ range .Header.Parameters }}* {{ .Name }}: {{ if .IsArray }}array of {{ end }}{{ .Type }}
+{{ range .Header.Parameters -}}
+* {{ .Name }}: {{ if .IsArray }}array of {{ end }}{{ .Type }}{{ if .Description }}{{ "  " }}
```

Comment:
nit: Please remove the `{{ "  " }}` as it adds empty spaces to the end of the line.

Reply:
@jalseth that was intentional since some markdown renderers won't render description on next line unless previous line ends with two spaces

so this:
```
* foo string
  some string parameter -- without spaces
* foo string  
  some string parameter -- with spaces
```

renders as:
* foo string some string parameter -- without spaces
* foo string
  some string parameter -- with spaces

another option would be to have an empty line separating parameter and description, but that puts them in different paragraphs, which is also undesirable.

some markdown implementations also support `\` (backslash) instead of two trailing spaces, and `<br>` is also an option.

which one should I pick? backslash, trailing spaces, `<br>` tag, or an empty line?

Or maybe they should just be one one line separated visually with text styling or `--`?

Reply:
I think I would prefer an empty line, which is common when writing markdown. I have not seen this before any many linters will automatically remove trailing spaces at the end of lines, which is why I recommended this change.

Reply:
@jalseth I've ended up using `\` instead of empty line due to empty line putting description into different paragraph
If you still prefer empty line over trailing backslash – please ping me and I'll update the code.

Reply:
I would prefer the empty line or a `<br>`. It is the most common. The backslash [may not be supported](https://www.markdownguide.org/basic-syntax/#line-break-best-practices) by all Markdown parsers and as it is uncommon, its intent may not be known to humans reading the markdown without rendering.